src/ray/common/client_connection.cc

```diff
@@ -30,6 +30,7 @@
 
 #include "ray/common/event_stats.h"
 #include "ray/common/ray_config.h"
+#include "ray/util/process.h"
```

Comment:
this is a little bit of an odd dependency to introduce -- IMO instead let's split `SetFdCloseOnExec` out into a utility that is depended on by both this and `process.cc`

Reply:
Where do you think is a better place to put this utility function? I originally wanted to put it in `src/ray/util/util.h`, but I found that `src/ray/util/util.cc` includes `"ray/util/process.h"`, so this creates a circular dependency between `process.h` and `util.h`.

Reply:
I'm not very familiar with Bazel. Do we perhaps have a way to handle circular dependencies in Bazel?

Reply:
I see... wouldn't expect that `util.h` requires `process.h`. Do you know what it's including from there?

Reply:
https://github.com/ray-project/ray/blob/8f7e30a2beb3003d09392b2f89a6d1a99b7ac338/src/ray/util/util.cc#L201

`IsProcessAlive` and `IsParentProcessAlive`